src/tribler/gui/debug_window.py

```diff
@@ -641,7 +640,7 @@ def load_events_tab(self):
         for event_dict, timestamp in tribler_received_events:
             item = QTreeWidgetItem(self.window().events_tree_widget)
             item.setData(0, Qt.UserRole, event_dict)
-            item.setText(0, f"{event_dict['type']}")
+            item.setText(0, f"{event_dict['topic']}")
```

Comment:
**Nitpick, noncritical:** probably it is safer to use `get` here...

```suggestion
            item.setText(0, f"{event_dict.get('topic')}")
```

Reply:
I agree